packages/react/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@logto/react",
+  "version": "0.1.0",
+  "source": "src/index.ts",
+  "main": "lib/index.js",
+  "exports": "./lib/index.js",
+  "typings": "lib/index.d.ts",
+  "license": "MPL-2.0",
+  "scripts": {
+    "dev:tsc": "tsc -p tsconfig.build.json -w --preserveWatchOutput",
+    "preinstall": "npx only-allow pnpm",
+    "precommit": "lint-staged",
+    "build": "tsc -p tsconfig.build.json",
+    "lint": "eslint --ext .ts --ext .tsx src",
+    "package": "parcel build",
```

Comment:
why do we need parcel here if we just use `tsc` for compiling?

Reply:
Removed parcel for now.